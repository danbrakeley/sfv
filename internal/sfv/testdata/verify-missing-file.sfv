; sfv testdata
verify-1.bin 0D532A24
verify-2.bin 4EB2B222
verify-3.bin 4EB2B222
