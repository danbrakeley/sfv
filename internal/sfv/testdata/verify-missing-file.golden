0 223554084 223554084 verify-1.bin <nil>
1 1320333858 1320333858 verify-2.bin <nil>
2 1320333858 0 verify-3.bin CreateFile testdata\verify-3.bin: The system cannot find the file specified.
0/28 verify-1.bin
8/28 verify-2.bin
0 223554084 223554084 verify-1.bin <nil>
1 1320333858 1320333858 verify-2.bin <nil>
2 1320333858 0 verify-3.bin CreateFile testdata\verify-3.bin: The system cannot find the file specified.
